zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/main/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProvider.java

```diff
@@ -70,6 +99,11 @@
     private Server server;
     private final MetricsServletImpl servlet = new MetricsServletImpl();
     private final Context rootContext = new Context();
+    private final int numWorkerThreads = Integer.getInteger(ZOOKEEPER_PROMETHEUS_NUM_WORKER_THREADS,
```

Comment:
we should read these configuration parameters in the "configure" method
this way we are not depending on system properties

Reply:
I see. I will make the change so they will be read from configuration file just the other properties such as `httpHost`

We also need to document them in the zookeeperAdmin.md, right?